app/views/assessments/_install_assessment_table_js.html.erb

```diff
@@ -0,0 +1,70 @@
+<% content_for :javascripts do %>
+  <script>
+      let importAsmtsUrl = new URL(`<%= url_for(importAssessments_course_assessments_url(@course)) %>`)
+      // dropdown toggle
+      function toggleInstallDropdownOptions(dropdownId, tableId) {
+          if (!$(`#${tableId}`).parents().hasClass('import-incomplete')) {
+              toggleOptions(`#${dropdownId}`, `#${tableId}`);
+          }
+      }
+      $(function() {
+          $("#installSelected").click(function(e) {
+              const msg = "Are you sure you want to import the selected assessments?";
+              if (!window.confirm(msg))
+                  return false;
+
+              const $btn = $(e.target);
+              $btn.text("Importing...");
+              $btn.addClass("disabled");
+              $("#unselectAll").addClass("disabled");
+              $("#selectAll").addClass("disabled");
+
+              let checkedAsmts = $(".cbox").filter(':checked');
+              let checkedAsmtNames = $.map(checkedAsmts, function(cbox){
+                  return $(cbox).attr('name')
+              })
+              $.each(checkedAsmts, (i, cbox) => {
+                  $(cbox).prop("disabled", true)
+              })
+              $.post(importAsmtsUrl, {assessment_names: checkedAsmtNames}, function(data){
+                  const errored = data.filter((entry) => entry.status === "FAIL").length > 0;
+                  if (errored) {
+                      $('#error-install-help').show()
+                  }
+                  $.each(checkedAsmts, (i, cbox) => {
+                      $(cbox).closest(".import-incomplete").removeClass("import-incomplete").addClass("import-complete")
```

Comment:
nit: not really sure how robust `closest` is. It might be easier to assign the relevant elements classes/ids based on the assessment name then query for the exact elements here.